### DISCUSSION TITLE
ReadMe.md showing instead of index.html

### DISCUSSION THREAD

**Discussion by rezy12** [community/community#106646] [Programming Help]
Hello I am new to react and was wondering why my website page shows the read.me instead of my index.html file. I have already tried the switching of branches to `gh-pages` `settings `pages branch to `gh-pages` `root ` `save `but it is still hosting the readme.md. I posted this question earlier but had the response of--- "your file index.html is in a subfolder called public, and not directly inside the root directory. This placement is important for GitHub Pages to recognize it as the main page of your site" but I tried to rearrange it but still same problem: my code is:

https://github.com/rezy12/BlisfulBites

and here is the layout: I don't know the layout it should be where should I put the index.html file Thank you in advanced:

![Screenshot 2024-02-07 101336](https://github.com/community/community/assets/45492208/2486ad10-44c6-4dd1-a77b-c2acefd2ee00)

**Comment by gil-son** [comment]
Hello,

try to remove the extension .bak:

README.md

**Reply by rezy12** [reply]
same problem, I tried to rename it to different type of file (saw it on overstack) but still shows readme.md

**Reply by gil-son** [reply]
Verify that [doc](https://blog.logrocket.com/deploying-react-apps-github-pages/) and certain to check those settings

or that [video](https://www.youtube.com/watch?v=7wzuievFjrk)

**Reply by gil-son** [reply]
Save your README.md in other directory

And execute the project, maybe other problem will appear or the problem will be solved

**Reply by rezy12** [reply]
So I tried to move it around in different directories but had the `404 error page` so I think it must be in the right location correct? I have another project that works fine do you think I can clone it to this repository and make the same changes would that work? For example if i git clone to this repository and make the follow layout changes it would work because the repository of the other page has extra directory show below (btw don't remember how I added those folders in terminal command):

![Screenshot 2024-02-07 101336](https://github.com/community/community/assets/45492208/b14b4673-0737-4f5e-9e66-e6727c75911b)

**Comment by offensive-vk** [comment]
The `index.html` file should be in `/public/` directory, since its a static file.
Remove that `.bak` extension and rename the filename to `readme.md` and move it to the root of the project.

**Reply by rezy12** [reply]
the thing is I already tried that I took it out of the public folder and recompiled it `npm run build`, `npm deploy`, but the same thing happened when I try to host the website. Do you think it can be a setting in Github instead because i already tried switching page branches earlier?

**Reply by rezy12** [reply]
also renamed `ReadMe ` it as well but same error
